source/extensions/omni.isaac.lab/omni/isaac/lab/managers/observation_manager.py

```diff
@@ -83,13 +122,22 @@ def active_terms(self) -> dict[str, list[str]]:
         return self._group_obs_term_names
 
     @property
-    def group_obs_dim(self) -> dict[str, tuple[int, ...]]:
-        """Shape of observation tensor in each group."""
+    def group_obs_dim(self) -> dict[str, tuple[int, ...] | list[tuple[int, ...]]]:
+        """Shape of observation tensor in each group.
+
+        If the terms are concatenated, the value is a single tuple representing the shape of the
+        concatenated observation tensor. Otherwise, the value is a list of tuples, where each tuple
+        represents the shape of the observation tensor for a term in the group.
+        """
         return self._group_obs_dim
 
     @property
     def group_obs_term_dim(self) -> dict[str, list[tuple[int, ...]]]:
-        """Shape of observation tensor for each term in each group."""
+        """Shape of observation tensor for each term in each group.
+
+        The value is a list of tuples, where each tuple represents the shape of the observation tensor
```

Comment:
Similarly here, can you describe the keys (observation group name I believe) too?

Reply:
Fixed.